awx/api/templates/api/bulk_host_create_view.md

```diff
@@ -0,0 +1,16 @@
+# Bulk Host Create
+
+This endpoint allows the client to create multiple hosts and associate them with an inventory. They may do this by providing the inventory ID and a list of json that would normally be provided to create hosts.
+
+Example:
+
+```
+{
+"inventory": 1,
+"hosts": [
+    {"name": "example1.com"},
+    {"name": "example2.com"}
+]
+
+}
+```
```

Comment:
I would really like to see info about the returned data structure, and this seems like the most appropriate place. In the returned data, for instance, you provide a "url" key, but it doesn't have the full ordinary Host serializer, almost certainly due to performance considerations. That really needs to be spelled out to the user from the get-go, because I find it very likely that people will get the response data and do something with it. This could be done with an example, that's fine. It might be good to include extra_vars, because vars format is always a pain.

Reply:
Sure, I do have an example of how to use variables in the docs section of the bulk_api feature. But I will add it here too.